Improbable/Postgres/Improbable.Postgres/MetricsPusher.cs

```diff
@@ -56,16 +63,11 @@ private void Scrape()
 
         public void StartPushingMetrics(TimeSpan interval)
         {
-            metricsPusherTask = Task.Run(() => Run(interval, metricsCancellationTokenSource.Token));
-        }
+            metricsCancellationTokenSource = new CancellationTokenSource();
 
-        public void StopPushingMetrics()
-        {
-            metricsCancellationTokenSource?.Cancel();
-            metricsPusherTask?.Wait();
+            metricsPusherTask = Task.Run(() => Run(interval, metricsCancellationTokenSource.Token));
```

Comment:
`Task.Run()` has an overload that takes a task cancellation token as well, do we want to use that too?